app/views/admin/users/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> 
+  <%= @user.firstname %> 
+  <%= @user.lastname %>
+</h1>
+<p> <%= @user.email %> </p>
+<small> <%= @user.created_at %> </small>
```

Comment:
Please add new line at the end of this file.

Reply:
there was already a new line in it.